pkg/JuliaInterface/PackageInfo.g

```diff
@@ -15,6 +15,24 @@ Date := "07/01/2021", # dd/mm/yyyy format
 License := "GPL-2.0-or-later",
 
 Persons := [
+  rec(
+    LastName := "Breuer",
+    FirstNames := "Thomas",
+    IsAuthor := true,
+    IsMaintainer := true,
+    Email := "sam@math.rwth-aachen.de",
+    WWWHome := "http://www.math.rwth-aachen.de/~Thomas.Breuer",
+    PostalAddress := Concatenation( [
+                       "Thomas Breuer\n",
+                       "Lehrstuhl für Algebra und Zahlentheorie\n",
+                       "RWTH Aachen\n",
+                       "Pontdriesch 14/16\n",
+                       "52062 Aachen\n",
+                       "Germany"
+      ] ),
+    Place := "Aachen",
+    Institution := "RWTH Aachen",
+  ),
   rec(
     IsAuthor := true,
     IsMaintainer := true,
```

Comment:
```suggestion
    IsMaintainer := false,
```

Reply:
@mohamed-barakat Perhaps I am misunderstanding what "maintainer" means; I thought that an author is somebody who has contributed to the package, perhaps some time ago, and that a maintainer is somebody who is still working on it.

Reply:
And Sebastian is no longer working on it

Reply:
Ah, now I understand. Sorry for the confusion.